fetchData.js

```diff
@@ -0,0 +1,114 @@
+class Digimon {
+    constructor(element, name, img, level) {
+        this.element = element;
+        this.name = name;
+        this.img = img;
+        this.level = level;
+        let setLevel = level.toLowerCase();
+        if (setLevel != 'in training') {
+            this.element.setAttribute('class', `digimon ${setLevel}`)
+        } else {
+            this.element.setAttribute('class', 'digimon fresh')
+        }
+    }
+
+    renderElement() {
+        this.element.innerHTML = '';
+
+        const nameElement = document.createElement('p');
+        nameElement.classList.add('digimon_name');
+        nameElement.innerText = this.name;
+
+        const imgElement = document.createElement('img');
+        imgElement.src = this.img
+
+        const levelElement = document.createElement('p');
+        levelElement.classList.add('level');
+        levelElement.innerText = 'Level: ' + this.level;
+
+        this.element.appendChild(nameElement);
+        this.element.appendChild(imgElement);
+        this.element.appendChild(levelElement);
+    }
+}
+
+async function fetchData() {
+    const response = await fetch('https://digimon-api.vercel.app/api/digimon');
+    if (response.ok) {
+        const data = await response.json();
+        return data
+    }
+    return [];
+}
+
+fetchData()
+    .then((data) => {
+        const holder = document.querySelector('.holder');
+        holder.innerHTML = '  ';
+        for (let i = 0; i < data.length; i++) {
+            const digimonEl = document.createElement('div');
+            const digimonElement = new Digimon(digimonEl, data[i].name, data[i].img, data[i].level);
+            digimonElement.renderElement();
+            holder.appendChild(digimonEl);
+        }
+    })
+
+async function PassName(name) {
```

Comment:
A nitpicky thing: your function names should always start with a lowercase letter (classes start with uppercase letter). Just a common practice amongst developer.